### PR TITLE
Added a Kotlin section in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Redirion**
#### What is it?
- [ ] Bugfix (user facing)
- [ ] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [X] Meta improvement to the project (dev facing)

#### Description of the changes in your PR
Core team does not want to convert to Kotlin yet and sees Java as the easier to learn and more well adopted language.

This stance might of course change in the future. For example it could be reasonable to do a complete transition to Kotlin once it is decided that the minSdk is raised to 21 or higher, as we then could use Jetpack particularly Lifecycle and Compose.

#### Due diligence
- [X] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -39,6 +39,11 @@ You'll see *exactly* what is sent, be able to add **your comments**, and then se
 * Create PRs that cover only **one specific issue/solution/bug**. Do not create PRs that are huge monoliths and could have been split into multiple independent contributions.
 * NewPipe uses [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor) to fetch data from services. If you need to change something there, you must test your changes in NewPipe. Telling NewPipe to use your extractor version can be accomplished by editing the `app/build.gradle` file: the comments under the "NewPipe libraries" section of `dependencies` will help you out.
 
+### Kotlin in NewPipe
+* NewPipeExtractor shall remain Java only
```
Comment (mhmdanas) [inline .github/CONTRIBUTING.md:43]: I think this should go in the extractor's CONTRIBUTING.md, not here.

Comment (Redirion) [reply .github/CONTRIBUTING.md:43]: the extractor has no contributing.md and is also mentioned here, so I thought it would be fitting here

Comment (mhmdanas) [reply .github/CONTRIBUTING.md:43]: GitHub does not show a link to this when opening PRs on the extractor, so I think this still does not fit here and that a CONTRIBUTING.md should be added there.

**Comment by XiangRongLin** [comment]
I agree with the point made on IRC that just simply converting classes to kotlin in not valueable. I would go even further and say it's harmfull, because nearly no part is covered by tests. So these changes rely on use trusting the contributor or on the time of the reviewer to verify that no bugs are introduced.

Whats missing for me is the stance on new classes.  
I personally love Kotlin and would like to at least write new files in kotlin in order to take advantage of all it's features.

**Review by litetex — APPROVED** [review]
LGTM

**Comment by litetex** [comment]
@mhmdanas 
Despite me approving the PR, could you also check it again?
You have been requested primarily as reviewer 😊

**Review by mhmdanas — APPROVED** [review]

**Review by Stypox — APPROVED** [review]

**Pull request merged**